release-notes/elastic-security/known-issues.md

```diff
@@ -19,15 +19,3 @@ Known issues are significant defects or limitations that may impact your impleme
 % Workaround description.
 
 :::
-
-:::{dropdown} Duplicate alerts can be produced from manually running threshold rules
```

Comment:
@benironside I removed these since they're already [doc'd in the feature docs](https://docs-v3-preview.elastic.dev/elastic/docs-content/pull/1014/solutions/security/detect-and-alert/manage-detection-rules#manually-run-rules) as things to be aware of.

Reply:
Added in [34daf1a](https://github.com/elastic/docs-content/pull/1014/commits/34daf1aed024c4ee5214e4a1e6a8440fd6c5828e).